src/sentry/options/defaults.py

```diff
@@ -1586,6 +1586,12 @@
     default=100,
     flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
 )
+register(
+    "statistical_detectors.query.transactions.timeseries_days",
+    type=Int,
+    default=14,
+    flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
+)
```

Comment:
you might want to register the option first in a separate PR. I don't know for sure but I think reading option's value in the same PR as registering it could cause some issues